### DISCUSSION TITLE
XTB python interface charge and mulitplicity?

### DISCUSSION THREAD

**Discussion by woodg07** [grimme-lab/xtb#541] [General]
Dear Forum,

I was trying to find documentation on entering Charge and multiplicity information using the python interface i.e.:

    from xtb.interface import Calculator, Param
    calc = Calculator(Param.GFN2xTB, atomic_nums, positions) # is there a key word here?

It seems that the standalone binary has this capability:

https://github.com/grimme-lab/xtb_docs/blob/master/source/sp.rst#charge-and-multiplicity

Thanks!

**Comment by awvwgk** [comment]
The constructor for the `Calculator` object can take a `charge` and an `uhf` keyword argument to set those properties (see https://xtb-python.readthedocs.io/en/latest/general-api.html#xtb.interface.Molecule).
